base/scss/overrides/reset/_civicrm.scss

```diff
@@ -120,9 +120,21 @@ table {
   width: 26px;
 }
 
+// Fixes the dropdown arrow width for crm form selects:
+.select2-container.crm-form-select {
+  .select2-arrow {
+    min-width: 24px;
+  }
+
+  .select2-arrow b {
+    margin-top: 0;
+  }
+}
+
 // removes fixed width for inputs added by CiviCRM:
 input.crm-form-text {
-  min-width: inherit;
+  @extend .form-control;
+  width: initial !important;
```

Comment:
Initial means 100% in our case?

Reply:
`width: initial` does not mean `width: 100%`.

This is how things will work after merging this change:
![crm-form-text](https://user-images.githubusercontent.com/1642119/30374484-96b9af84-9831-11e7-8df2-2163a107a3fe.png)

`.crm-form-text` will behave exactly how they were before: they'll have a default width (depends on the browser), but if a size attribute is specified it will adjust to that size.

`.form-control` take 100% width of their container, even if a size property is defined. This is good for responsive behaviour (which this admin is not), but it requires extra work if you want to adjust the input size. You'd either add a css style to it, or wrap it in a `col-*` div.

A middle ground can be reached though: we could leave inputs to be `width: 100%` except when they have the size attribute.

**Side notes:**
* form inputs without a class are not visible (nothing to do with this PR)
* even 100% does not mean 100% since there's a definition on Shoreditch's Bootstrap to make form-control inputs no longer than 370px

Reply:
Update:

I have made so all `.crm-form-text` extend from `.form-control`, but only the ones with the size property will have `width: initial`.